alpha/declcfg/diff.go

```diff
@@ -79,6 +79,19 @@ func (g *DiffGenerator) Run(oldModel, newModel model.Model) (model.Model, error)
 		return nil
 	}
 
+	// Prunes empty channels
+	pruneEmptyChannels := func() error {
```

Comment:
If this is only called once, in-line the code in that spot. Perhaps this could live in Includer.Run() too.

Reply:
Moved this down to where I was calling it. The only reason I didn't put it in `Includer.Run()` is because this is only an issue when `IncludeAdditively` is false I believe.